hi-ml-azure/src/health_azure/himl.py

```diff
@@ -648,6 +635,8 @@ def submit_run(workspace: Workspace,
         the completion of this run (if True).
     :param wait_for_completion_show_output: If wait_for_completion is True this parameter indicates whether to show the
         run output on sys.stdout.
+    :param display_name: The name that will be displayed in the AML UI. If not provided, the unique run ID will be
+        used as the display name.
```

Comment:
Does this mean we don't get the random AML names e.g. `hungry_beaver`?

Reply:
you're right, i have  not phrased that correctly

Reply:
fixed

Reply:
Phew. I really enjoyed those :D